src/lib/seq2HLA.ml

```diff
@@ -0,0 +1,28 @@
+
+open Common
+open Run_environment
+module K = KEDSL
+
+let hla_type ?host ~work_dir ~run_with ~r1 ~r2 ~run_name =
```

Comment:
We can add this in another PR, but might be nice to expose configuration (if Seq2HLA has any) to the user.